src/components/Settings.vue

```diff
@@ -4,35 +4,35 @@
     <v-card-text>
       <OpenImage @change="onImageOpen"
         @delete="ImageDelete"
-        :ImageCheck = "ImageCheck"/>
-      <ImageProcessor :imageUrl = "imageUrl" @imageSampled="onImageSampled"/>
+        :imageCheck="imageCheck"/>
+      <ImageProcessor
+        :image-width="imageWidth"
+        :image-height="imageHeight"
+        :image-url="imageUrl"
+        @imageSampled="onImageSampled"/>
       <v-layout row wrap>
           <v-flex xs3 style="margin: 24px;">
               <v-text-field
               label="Width"
               id="widthSetting"
-              placeholder="32"
+              placeholder="64"
```

Comment:
The input will never be empty, so we don't really need a placeholder

Reply:
Changed. Removed the 'placeholder' value altogether.